### ISSUE TITLE
[Bug] <Short description>

### ISSUE THREAD

**Issue by TDosYos** [PojavLauncherTeam/PojavLauncher_iOS#732]
### Describe the bug

In the process of playing, often without warning card died in a picture. At this point, you can't do anything but you can still tell by the audio that the game is running.
The report that appears in the log at this time:
[09:42:05] [ Render thread/INFO]: OpenGL debug message: id=1280, source=API, type=ERROR, severity=HIGH, message='Enum 0x8501 is currently not supported.'

### The log file and images/videos

![34AEB7CC-C0EE-4D1A-89F0-664EFF79ACD1](https://github.com/user-attachments/assets/99416bdb-8293-4e56-aba9-a8b5e1d500f0)
![541A7405-A4CC-456E-91CC-62558D4784B4](https://github.com/user-attachments/assets/633ae250-e397-40fb-b1c8-1e7796bf2776)
![058B5FDB-86D3-49BF-AA77-83775E2338A2](https://github.com/user-attachments/assets/59721119-12d8-4952-a5ea-73a60cc194d1)

### Steps To Reproduce

```markdown
1.start the pojavlauncher
2.launcher the game
3.An error occurred after entering the world.
```

### Expected Behavior

Hope it will be fixed

### Platform

```markdown
- Device model: iPhone 7 Plus
- iOS version: 15.8.2
- PojavLauncher version: 
Development build #1188
```

**Comment by TDosYos** [comment]
Renderer used:ANGLE
Minecraft version: 1.21 + fabric

**Comment by Kiokingz** [comment]
the game still runs fine, correct?
if so, just don't open output log, ignore it

**Comment by TDosYos** [comment]
> the game still runs fine, correct?
> 
> if so, just don't open output log, ignore it

No, actually it's not. It just gets stuck in one picture and can't do anything.

**Comment by CNjimmy6895** [comment]
It seems to be a problem with the redstone structure.
